test/heatmap.test.js

```diff
@@ -78,7 +78,7 @@ describe('Heatmaps', () => {
     });
   });
 
-  describe('Visualization', () => {
```

Comment:
Disabled this one test which is getting back "error" data from `resgen.io`:

- `https://resgen.io/api/v1/tileset_info/?d=Bu6Djvt9T_mgZPHLig3NfQ`

```
{"Bu6Djvt9T_mgZPHLig3NfQ": {"error": "Unknown filetype ", "name": "87K x 87K pairwise genome comparison", "coordSystem": ""}}
```

Reply:
That endpoint is fixed on resgen.io now.